main.go

```diff
@@ -379,6 +405,36 @@ func main() {
 	bucket := lookupEnv("COLLECT_INFLUXDB_BUCKET", dryRun)
 	measurement := lookupEnv("COLLECT_INFLUXDB_MEASUREMENT", dryRun)
 
+	var mqttClient MQTT.Client
+	mqttServerAddress, ok := os.LookupEnv("COLLECT_MQTT_SERVER")
+	if ok && !dryRun {
+		mqttPort, isMqttPortSet := os.LookupEnv("COLLECT_MQTT_PORT")
+		if !isMqttPortSet {
+			mqttPort = "1883"
+		}
+		mqttUsername := lookupEnv("COLLECT_MQTT_USERNAME", dryRun)
+		mqttOpts := MQTT.NewClientOptions().AddBroker("tcp://" + mqttServerAddress + ":" + mqttPort)
+		mqttOpts.SetUsername(mqttUsername)
+		mqttPassword, isMqttPasswordSet := os.LookupEnv("COLLECT_MQTT_PASSWORD")
+		if isMqttPasswordSet {
+			mqttOpts.SetPassword(mqttPassword)
+		}
+		mqttOpts.SetAutoReconnect(true)
+		mqttOpts.SetClientID("rtlamr-collect-" + strconv.Itoa(rand.Intn(100)))
```

Comment:
Is this necessary?

Reply:
It certainly can be a set of config variables, but this seems to me to be alright as sane defaults.

Client ID will either be set to some random value or this, which should simplify debugging on MQTT side of things.